pkg/operatorrules/registry.go

```diff
@@ -51,7 +53,10 @@ func ListRecordingRules() []RecordingRule {
 	}
 
 	slices.SortFunc(rules, func(a, b RecordingRule) int {
-		return cmp.Compare(a.GetOpts().Name, b.GetOpts().Name)
+		aKey := a.Alert + ":" + a.Expr.String()
```

Comment:
you copied this from the alert, and kept the alert, instead of using the name

Reply:
right sorry did it late last night..

Reply:
fixed